base/silent/src/main/java/org/openscience/cdk/silent/Polymer.java

```diff
@@ -64,7 +63,7 @@
      */
     public Polymer() {
         super();
-        monomers = new Hashtable<String, IMonomer>();
+        monomers = new HashMap<>();
```

Comment:
This is new Java magic, right? It takes the typing automatically from the type of the 'monomers' variable definition, correct? (I guess a rhetorical question :)